src/content/pages/index.mdx

```diff
@@ -60,12 +63,11 @@ Whether you're taking notes in another app, reviewing a prototype in Figma, or j
 
 <Image alt="The Mic Drop preferences window." src="images/customisable.png" />
 
-<TextBlock>
+<TextBlock end>
 
 ## Customisable
 
-Sure, most video apps have their own keyboard shortcuts, but they're all different. Instead of trying to remember different shortcuts for each app, Mic Drop allows you to create a custom control that makes sense to you. You can even use a double-tap if you're so inclined!
-
+Sure, most video apps have their own keyboard shortcuts, but they're all different. Instead of trying to remember different shortcuts for each app, Mic Drop allows you to create a custom control that makes sense to you.
```

Comment:
```suggestion
Sure, most video apps have their own keyboard shortcuts, but they're all different. Instead of trying to remember different shortcuts for each app, Mic Drop allows you to create a custom control that makes sense to _you_.
```

thoughts?